app/models/manageiq/consumption/showback_event/common.rb

```diff
@@ -0,0 +1,22 @@
+module ManageIQ::Consumption::ShowbackEvent::COMMON
+
+  def get_metric(type)
+    if is_container?
+      resource.vim_performance_states.last.state_data[type.to_sym] || 0
```

Comment:
Could you comment this code?
 It should be clear why we are doing this so it can be refactored in the future